javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -53,6 +53,9 @@ function AppNavbar() {
           <NavDropdown.Item as={Link} to="/statistics/classSize">
             Average Class Size by Department
           </NavDropdown.Item>
+            <NavDropdown.Item as={Link} to="/statistics/totalCourses">
+                Total Courses By Department
+            </NavDropdown.Item>
```

Comment:
Very minor: it would be good to adjust the indents on this block so that it matches with the other tabs in the dropdown.

Reply:
Just fixed it